.pipelines/clang-format-ci.yml

```diff
@@ -0,0 +1,56 @@
+name: $(Year:yy).$(Month).$(DayOfMonth).$(rev:r)
+
+pr:
+  branches:
+    include:
+    - main
+
+pool:
+  vmImage: ubuntu-latest
+
+steps:
+- task: NodeTool@0
+  inputs:
+    versionSpec: '14.x'
+  displayName: 'Install Node.js'
+
+- script: |
```

Comment:
nit: you don't seem to need '|', you can put the next line after ':'

Reply:
If you edit the pipeline from the web UI, and add a inline script, you will found the `: |`.
no change will be made since I use the same format in this file